modules/gdscript/gd_editor.cpp

```diff
@@ -2393,6 +2394,14 @@ Error GDScriptLanguage::complete_code(const String &p_code, const String &p_base
 
 void GDScriptLanguage::auto_indent_code(String &p_code, int p_from_line, int p_to_line) const {
 
+	bool use_space_indentation = EDITOR_DEF("text_editor/indent/indent_using_spaces", false);
+	int indent_size = EDITOR_DEF("text_editor/indent/tab_size", 4);
+
+	String space_indent = "";
+	for (int i = 0; i < indent_size; i++) {
+		space_indent += " ";
+	}
+
```

Comment:
I would make this a function, like `_get_indentation` (which would return a tab or spaces), so that it can be re-used.